test/danetest.c

```diff
@@ -192,7 +192,7 @@ static ossl_ssize_t hexdecode(const char *in, void *result)
     for (byte = 0; *in; ++in) {
         int x;
 
-        if (isspace(_UC(*in)))
+        if (ossl_isspace(_UC(*in)))
```

Comment:
Shouldn't the _UC() - which is really just an `unsigned char` cast, be removed here?